task/redhat-codeready-dependency-analysis/0.1/samples/post-task-setup.yaml

```diff
@@ -0,0 +1,32 @@
+# A Custom task given for demo purpose to read the report and make decision about build
+apiVersion: tekton.dev/v1beta1
+kind: Task
+
+metadata:
+  name: post-task-setup
+spec:
+  params:
+    - name: output-file-path
+      default: redhat-codeready-dependency-analysis-report.json
+    - name: image
+      default: ghcr.io/fabric8-analytics/crda-python:3.6
```

Comment:
nit: 3.6 will be out of support by end of year, any plans for updating it ?

Reply:
Okay, I will do one thing for it I will make Python 3.7 as default, but as I said in my previous comment it is just a default image and it is not at all mandatory for CRDA to use 3.6 only, we are proving list of images for all supported ecosystems and all available versions of those ecosystem and we are asking users to choose any one of them and give that image as an argument to pipeline. If a user still wants to run task on python 3.6, we will run it and if there is some issue users need to change image tag to 3.7/3.8/../..

Reply:
Default image tag is updated to 3.7